src/pages/Winners2021.js

```diff
@@ -0,0 +1,369 @@
+import React from "react"
```

Comment:
considering making this page something like /pages/2021/winners instead of WiNneRS2k21 :